src/webui/api/synccontroller.cpp

```diff
@@ -445,6 +445,14 @@ void SyncController::maindataAction()
 
     data["categories"] = categories;
 
+    QVariantList tags;
+    const auto &tagsList = session->tags();
+    for (auto it = tagsList.cbegin(); it != tagsList.cend(); ++it) {
+        tags << *it;
+    }
```

Comment:
and omit curly braces for single line `for` body statements such as this one and at other places too.